regulations/static/regulations/css/less/main.less

```diff
@@ -41,6 +41,8 @@ Modules
 @import "module/error.less";
 @import "module/paragraph-markers.less";
 @import "module/comment.less";
+@import "module/comment-review.less";
+@import "module/comment-media-queries.less";
```

Comment:
Not for now, but would it make sense to use a subdir rather than hyphen separation?